jenkins-automation.groovy

```diff
@@ -172,6 +175,64 @@ behave -k -f=plain --logging-level=INFO --junit --junit-directory=test-results \
   }
 }
 
+
+// Requires: 
+// - BASE_URL: the starting url of your application
+//
+job('template-bdd-security'){
+  using 'template-base'
+
```

Comment:
Yes, I'd like us to for sure, but went with the old style since we haven't fully formed the idea.